src/libsbn.hpp

```diff
@@ -392,6 +392,34 @@ TEST_CASE("libsbn: likelihood and gradient") {
   }
 }
 
+#include "rooted_sbn_instance.hpp"
```

Comment:
This should be moved to `rooted_sbn_instance.hpp` -- there is a "dummy" test there that you can replace.